test/ShowTeamModal.test.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * @vitest-environment jsdom
+ */
+
+import { expect, vi, test } from "vitest"
+import { render, screen, fireEvent } from "test/utils"
+import { ShowTeamModal } from "src/assignments/components/ShowTeamModal"
+
+test("renders show team modal", async () => {
+  const curentTeam = {
+    id: 1,
+    name: "team1",
+    contributors: [
+      {
+        id: 1,
+        user: {
+          id: 1,
+          username: "user1",
+        },
+      },
+      {
+        id: 1,
+        user: {
+          id: 1,
+          username: "user2",
+        },
+      },
+    ],
+  }
+
+  render(<ShowTeamModal team={curentTeam}></ShowTeamModal>)
+
+  expect(await screen.getByText("team1")).toBeInTheDocument()
+  const openModalBtn = screen.getByTestId("open-modal")
+  expect(openModalBtn).toBeInTheDocument()
+  fireEvent.click(openModalBtn)
+  expect(await screen.findByText("user1")).toBeInTheDocument()
+  expect(await screen.findByText("user2")).toBeInTheDocument()
+  const closeModalBtn = screen.getByTestId("open-modal")
```

Comment:
The current tests are positive tests – they test what you would expect to be present. 

I recommend to also include negative tests – testing what you would expect to **not** be present.

```suggestion
  expect(await screen.findByText("user2")).toBeInTheDocument()
  expect(await screen.queryByText("user3")).not.toBeInTheDocument()
  const closeModalBtn = screen.getByTestId("open-modal")
```